news/654-fix_channel_remap

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* urls in generated envs.txt must use url given in channel_remap (was installer-build time url before)
```

Comment:
```suggestion
* URLs in generated `envs.txt` will now use the correct URL given in `channel_remap`, instead of the incorrect, installer-build-time only value. (#656)
```